editor/src/utils/feature-switches.ts

```diff
@@ -27,6 +29,7 @@ let FeatureSwitches: { [feature in FeatureName]: boolean } = {
   'Re-parse Project Button': !(PRODUCTION_CONFIG as boolean),
   'Performance Test Triggers': !(PRODUCTION_CONFIG as boolean),
   'Canvas Strategies Debug Panel': false,
+  'Nine block control': true,
```

Comment:
`TODO` before merge: set to false

Reply:
I think a feature switch for 1 extra inspector section is not really necessary